kDrive/SceneDelegate.swift

```diff
@@ -222,6 +229,17 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate, AccountManagerDel
         }
     }
 
+    @discardableResult
+    private func continueToWebActivityIfPossible(_ scene: UIScene, userActivity: NSUserActivity) async -> Bool {
```

Comment:
In the `continueToWebActivityIfPossible` the `to` implies that we go to the web. Instead we are coming from the web and we try to `continueWebActivity`

```suggestion
    private func continueWebActivityIfPossible(_ scene: UIScene, userActivity: NSUserActivity) async -> Bool {
```